### DISCUSSION TITLE
How do I know if a broadcast has been started ?

### DISCUSSION THREAD

**Discussion by volkankoc** [bluenviron/mediamtx#1582] [Questions]
How do I know if a broadcast has been started
How do I know when a broadcast has been terminated?
According to this event, I will enter information into the database.

**Comment by aler9** [comment]
Hello, you can use `runOnReady` to launch a custom script that calls two webhooks when a stream is started and when a stream is terminated.

Server configuration:

```yml
paths:
  all:
    runOnReady: /path/to/script/script.sh
```

script.sh:

```bash
#!/bin/bash
trap 'curl "http://my-server/stop?name=$RTSP_PATH"; [ -z "$(jobs -p)" ] || kill $(jobs -p)' INT
curl "http://my-server/start?name=$RTSP_PATH"
sleep 86400 &
wait
```

[a comment by an automated account is not shown]
